### ISSUE TITLE
[Documentation] iOS version should be specified in the docs

### ISSUE THREAD

**Issue by bdlukaa** [alexmercerind/media_kit#89]
`native_event_loop` require iOS 13 (See [this](https://github.com/alexmercerind/media_kit/blob/main/media_kit_native_event_loop/ios/media_kit_native_event_loop.podspec#L24)). This breaks apps that have iOS version < 13 specified.

### Proposal

Add `iOS` to the [Setup](https://github.com/alexmercerind/media_kit#setup) documentation. If possible, reduce the required iOS version or explain why such a high version is necessary. Usually, iOS 11 is necessary

**Comment by alexmercerind** [comment]
The version isn't even released yet. There's no documentation yet.

**Comment by bdlukaa** [comment]
I updated the request @alexmercerind . Pls have a look

**Comment by alexmercerind** [comment]
I will recommend you to experiment out. I can't say anything for iOS or macOS video embedding yet.

https://github.com/flutter/flutter/pull/96225 seems to only require iOS 9.

**Comment by alexmercerind** [comment]
@bdlukaa, goal is to release out a [new version 0.0.2](https://github.com/alexmercerind/media_kit/blob/code-cleanup-and-restructure/media_kit/CHANGELOG.md#002). There have been a number of advancements.
I'll be glad if you try out & pick the best suited minimum required version for the package before first release.

**Comment by birros** [comment]
@bdlukaa:
>  If possible, reduce the required iOS version or explain why such a high version is necessary.

When building libmpv for iOS, I had to set the min version of iOS to 13.0, impossible to compile for a lower version. 
This may be a mistake on my part, I will check, but I am 99% sure that support for a lower version is impossible.

**Comment by birros** [comment]
I confirm that the minimum supported iOS version is 13.0, the following error is from ffmpeg when targeting iOS 11.0:

```log
src/libavfilter/metal/utils.m:35:21: error: 'supportsFamily:' is only available on iOS 13.0 or newer [-Werror,-Wunguarded-availability-new]
        if ([device supportsFamily:MTLGPUFamilyCommon3]) {
                    ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS.sdk/System/Library/Frameworks/Metal.framework/Headers/MTLDevice.h:744:1: note: 'supportsFamily:' has been marked as being introduced in iOS 13.0 here, but the deployment target is iOS 11.0.0
- (BOOL)supportsFamily:(MTLGPUFamily)gpuFamily API_AVAILABLE(macos(10.15), ios(13.0));
^
src/libavfilter/metal/utils.m:35:21: note: enclose 'supportsFamily:' in an @available check to silence this warning
        if ([device supportsFamily:MTLGPUFamilyCommon3]) {
                    ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
src/libavfilter/metal/utils.m:35:36: error: 'MTLGPUFamilyCommon3' is only available on iOS 13.0 or newer [-Werror,-Wunguarded-availability-new]
        if ([device supportsFamily:MTLGPUFamilyCommon3]) {
                                   ^~~~~~~~~~~~~~~~~~~
/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS.sdk/System/Library/Frameworks/Metal.framework/Headers/MTLDevice.h:154:28: note: 'MTLGPUFamily' has been marked as being introduced in iOS 13.0 here, but the deployment target is iOS 11.0.0
typedef NS_ENUM(NSInteger, MTLGPUFamily)
                           ^
src/libavfilter/metal/utils.m:35:36: note: enclose 'MTLGPUFamilyCommon3' in an @available check to silence this warning
        if ([device supportsFamily:MTLGPUFamilyCommon3]) {
                                   ^~~~~~~~~~~~~~~~~~~
2 errors generated.
make: *** [libavfilter/metal/utils.o] Error 1
make: *** Waiting for unfinished jobs....
rm libavfilter/metal/vf_yadif_videotoolbox.metal.air libavfilter/metal/vf_yadif_videotoolbox.metallib
build step returned error code 2.
```

@bdlukaa Sorry 🙏
